src/contacts/ViewContact/ViewContactContainer.js

```diff
@@ -24,15 +25,16 @@ class ViewContactContainer extends Component {
     resources: PropTypes.object,
     baseUrl: PropTypes.string.isRequired,
     mutator: PropTypes.object,
-    // eslint-disable-next-line react/no-unused-prop-types
     orgId: PropTypes.string,
     showMessage: PropTypes.func.isRequired,
-    match: PropTypes.object,
+    match: ReactRouterPropTypes.match.isRequired,
+    history: ReactRouterPropTypes.history.isRequired,
+    location: ReactRouterPropTypes.location.isRequired,
   };
 
   static manifest = Object.freeze({
     contact: contactResource,
-    categories: categoriesResource,
+    contactCategories: categoriesResource,
```

Comment:
I believe you should use the same name of resource you want to re-use and the best way is to create a constant, something like `DICT_CATEGORIES` with value `vendorCategory`

Reply:
fixed

Reply:
it's not fixed - you've changed it to use parentResources, that is an anti-pattern.

Reply:
Maybe you've got the first comment wrong, but there is a clear words regarding a change to name of resource

Reply:
right, I'm sorry - looked at the new code change in ContactPeopleForm.js